### DISCUSSION TITLE
Filter Group Pattern Contribution

### DISCUSSION THREAD

**Discussion by corinne-nagel** [twilio-labs/paste#1601] [General] [answered]
Hi team✌️ I was wanting to focus on the filtering group pattern for think week that is on the list of prioritized patterns. I've already started some designs for this within Video and I've only just started a spec but would love to see how much I can get done in a week with a pattern. Is it possible to do a rapid pattern contribution of sorts in that timeframe? What would that look like?

Thanks!

**Comment by serifluous** [comment]
Yes let's do it! Rapid pattern contribution could definitely work. We need some guidelines and a design for a committee review, which we could try to do ad hoc during think week.

Alternatively, @katieporter and I were already talking about tackling filter groups as our next pattern since it's been such a huge need. You could take a first pass at the spec & design, and then we can pair after think week on the iterations & committee reviews.

**Reply by kate-porter** [reply]
Yesssss! 🙌 I like the idea of @corinne-nagel tackling the spec and design during think week, and then all of us partnering to take it through the committee afterwards. I think it'd probably be hard to get a committee review during think week.

**Reply by corinne-nagel** [reply]
Sweet! Let's do it! 🎉  I'll send a mini kick-off meeting for Tuesday with you two if that works and we can figure out a game plan/get the details nailed down.

**Comment by loreina** [comment, marked as answer]
A lil update: @corinne-nagel's handed off the pattern to DSYS. It's in our backlog to iterate on the design and finalize the spec.
